openedx_external_enrollments/tests/tests_factory.py

```diff
@@ -29,3 +39,24 @@ def test_get_enrollment_controller(self):
                 EdxInstanceExternalEnrollment,
             )
         )
+        controller = 'greenfig'
+        self.assertTrue(
+            isinstance(
+                ExternalEnrollmentFactory.get_enrollment_controller(controller),
+                GreenfigInstanceExternalEnrollment,
+            )
+        )
+        controller = 'icc'
+        self.assertTrue(
+            isinstance(
+                ExternalEnrollmentFactory.get_enrollment_controller(controller),
+                ICCExternalEnrollment,
+            )
+        )
+        controller = 'mit_hz'
+        self.assertTrue(
+            isinstance(
+                ExternalEnrollmentFactory.get_enrollment_controller(controller),
+                MITHzInstanceExternalEnrollment,
+            )
+        )
```

Comment:
I think you can use the DDT unpack feature to test different cases and avoid writing the same code over and over again. @ivanvgh 
https://ddt.readthedocs.io/en/latest/example.html

Reply:
Great, Taking this in consideration, thanks for the recommendation.